universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -23,6 +23,7 @@
 public class ApplicantData {
   private static final String EMPTY_APPLICANT_DATA_JSON = "{ \"applicant\": {}, \"metadata\": {} }";
   private static final Locale DEFAULT_LOCALE = Locale.US;
+  static final String REPEATED_ENTITY_NAME_KEY = "entity_name";
```

Comment:
Suggestion (blocking): this string already in appears in the codebase [here](https://github.com/seattle-uat/civiform/blob/47d7145e80098aa630fc8178396f61f34bcf410f/universal-application-tool-0.0.1/app/services/question/RepeaterQuestionDefinition.java#L108). Please create a shared constant to reference in both usages.

Reply:
good catch! fixed!